python/nammu/controller/AtfAreaController.py

```diff
@@ -182,3 +183,16 @@ def restore_highlight(self):
         length = len(self.getAtfAreaText())
         self.syntax_highlighter._highlight_match(0, length, Color.white)
         self.syntax_highlighter.syntax_highlight()
+
+    def getPositionFromLine(self, text, line_num):
+        '''
+        Given a block of text and a line number, return a tuple containing
+        the caret position at the start and end of the given line.
+        '''
+        compiled = re.compile(r"\n")
+        textiter = compiled.finditer(text)
+        if line_num != 1:
+            pos = [m.start() for m in textiter][line_num - 2:line_num]
+        else:
+            pos = [0, textiter.next().start()]
+        return pos
```

Comment:
Shouldn't this method return the position of the beginning of the new line instead of a pair with beginning and end? The previous code doing this needed so for the syntax highlight but in this case you just need to move the cursor to one specific position.